lib/rrd_updates.ml

```diff
@@ -192,6 +192,20 @@ let create_multi prefixandrrds start interval cfopt =
        (fun (_prefix,rrd) ->
           (* Find the rrds that satisfy the requirements *)
           Rrd.find_best_rras rrd pdp_interval cfopt start) prefixandrrds) in
+  let first_rra =
+    let rec go =
```

Comment:
This could be also implemented using `List.find_opt`
```
let first = rras |> List.find_opt (fun x -> x <> []) |> function  Some (x::_) -> x | _ -> raise No_RRA_Available
```